src/node/setupServer.ts

```diff
@@ -7,6 +7,7 @@ import { RequestHandlersList } from '../setupWorker/glossary'
 import { MockedRequest } from '../handlers/requestHandler'
 import { getResponse } from '../utils/getResponse'
 import { parseRequestBody } from '../utils/parseRequestBody'
+import isNode from '../utils/isNodeProcess'
```

Comment:
If it doesn't burden you, could you please use named export here?

```js
import { isNode } from '...'

// isNodeProcess.ts
export function isNode(...) {...}
```

It's a nitpick, but keeping up the code style is beneficial in a long run. Thanks for understanding!